gradle.properties

```diff
@@ -8,7 +8,7 @@ android.useAndroidX=true
 systemProp.org.gradle.internal.publish.checksums.insecure=true
 
 GROUP=com.squareup.workflow1
-VERSION_NAME=1.0.0-alpha18-SNAPSHOT
+VERSION_NAME=1.0.0-alpha18
```

Comment:
Your don't want this to be part of the changelog PR

Reply:
Why not? I thought we used to do that all the time.

Reply:
Reverted.